version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.13-alpha",
+  "version": "2.13",
```

Comment:
This will cause a CI build failure because [unstable dependencies exist](https://github.com/microsoft/vs-streamjsonrpc/blob/bd27af10e104685be469c799b6536808afb24d50/Directory.Packages.props#L24).